src/EditorFeatures/Core.Wpf/Options/EditorConfigDocumentOptionsProvider.cs

```diff
@@ -63,6 +65,69 @@ private void Workspace_DocumentOpened(object sender, DocumentEventArgs e)
             }
         }
 
+        private void Workspace_WorkspaceChanged(object sender, WorkspaceChangeEventArgs e)
+        {
+            switch (e.Kind)
+            {
+            case WorkspaceChangeKind.SolutionRemoved:
```

Comment:
Don't we indent cases? Not sure if bug or unstated .editorconfig rule.

Reply:
Unstated rule. Will be fixed in 32e150c9ae85d9ad3e3233ea374e3ae0cab26d3b (different PR).

Reply:
➡️ Amended the initial commit to fix the formatting.